draft-packages/menu/KaizenDraft/Menu/README.mdx

```diff
@@ -23,6 +23,10 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
 
 A menu provides access to additional actions and navigation links on demand while preserving a minimal interface.
 
+[Focus indicator](/guidelines/interaction-states)
+- On keyboard navigation show focus indicator on top selector.
+- For menu select options show focus indicator on keyboard navigation and on click (not hover).
```

Comment:
Should we put this inside the "Accessibility" heading? Or should it be it's own heading? (In which case, we should add `###` to make it a H3 for consistency)